### PR TITLE
Integrate tracefile writer into pipeline

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
**Reference to a related issue in the repository**
#52 

**Add a description**
- [x] Add tracefile writer to available FMUs in the pipeline
- [x] Create example integration test using tracefile writer
- [x] Create example python script to test recorded tracefile or alternatively use [OSI Validation](https://github.com/OpenSimulationInterface/osi-validation)

**Take this checklist as orientation for yourself, if this PR is ready for Maintainer Review**
- [x] My suggestion follows the [governance rules](https://github.com/openMSL/governance-and-documentation).
- [x] All commits of this PR are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
- [x] My changes generate no errors when passing CI tests. 
- [x] I updated all documentation (readmes incl. figures) according to my changes.
  Done in https://github.com/openMSL/sl-1-5-sensor-model-testing/pull/30
- [x] I have successfully implemented and tested my fix/feature locally.
- [x] Appropriate reviewer(s) are assigned.

### CODE REVIEW

**Comment by ClemensLinnhoff** [comment]
Tracefile writer works now. The tracefile is generated according to the [naming convention](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/interface/architecture/trace_file_naming.html) and stored as an artifact of the GitHub action.

For further processing of the tracefile in the workflow, we are waiting for several issues of the OSI validator:
- https://github.com/OpenSimulationInterface/osi-validation/issues/42
- https://github.com/OpenSimulationInterface/osi-validation/issues/43
- https://github.com/OpenSimulationInterface/osi-validation/issues/44
- https://github.com/OpenSimulationInterface/osi-validation/issues/45

**Comment by jdsika** [comment]
Add date+time to output file name?

**Comment by ClemensLinnhoff** [comment]
> Add date+time to output file name?

Date and time are already part of the file name, as specified in the official naming convention.

**Comment by ClemensLinnhoff** [comment]
> Tracefile writer works now. The tracefile is generated according to the [naming convention](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/interface/architecture/trace_file_naming.html) and stored as an artifact of the GitHub action.
> 
> For further processing of the tracefile in the workflow, we are waiting for several issues of the OSI validator:
> 
> * [Documentation not available and installation not working. OpenSimulationInterface/osi-validation#42](https://github.com/OpenSimulationInterface/osi-validation/issues/42)
> * [Error during install OpenSimulationInterface/osi-validation#43](https://github.com/OpenSimulationInterface/osi-validation/issues/43)
> * [Error during decompression of example trace file OpenSimulationInterface/osi-validation#44](https://github.com/OpenSimulationInterface/osi-validation/issues/44)
> * [Error during validation run OpenSimulationInterface/osi-validation#45](https://github.com/OpenSimulationInterface/osi-validation/issues/45)

Instead of using the OSI Validator, we created our own custom trace file analysis example.

File: test/integration/004_tracefile_analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# 004 Tracefile Analysis
+
+This test checks if an object inside the field of view of the sensor is detected by the model.
+
+## Scenario
+
+One object is placed in front of  the sensor. The x-coordinates of the objects in this scenario are:
```
[the remark this answers is not in the document]
Comment (ClemensLinnhoff) [reply test/integration/004_tracefile_analysis/README.md]: Ist gefixt.

**Review by LukasElster — APPROVED** [review]

**Pull request merged**